avm/res/recovery-services/vault/main.bicep

```
@@ -184,18 +187,10 @@ module rsv_replicationPolicies 'replication-policy/main.bicep' = [
     params: {
       name: replicationPolicy.name
       recoveryVaultName: rsv.name
-      appConsistentFrequencyInMinutes: contains(replicationPolicy, 'appConsistentFrequencyInMinutes')
-        ? replicationPolicy.appConsistentFrequencyInMinutes
-        : 60
-      crashConsistentFrequencyInMinutes: contains(replicationPolicy, 'crashConsistentFrequencyInMinutes')
-        ? replicationPolicy.crashConsistentFrequencyInMinutes
-        : 5
-      multiVmSyncStatus: contains(replicationPolicy, 'multiVmSyncStatus')
-        ? replicationPolicy.multiVmSyncStatus
-        : 'Enable'
-      recoveryPointHistory: contains(replicationPolicy, 'recoveryPointHistory')
-        ? replicationPolicy.recoveryPointHistory
-        : 1440
+      appConsistentFrequencyInMinutes: replicationPolicy.?appConsistentFrequencyInMinutes ?? 60
+      crashConsistentFrequencyInMinutes: replicationPolicy.?crashConsistentFrequencyInMinutes ?? 5
+      multiVmSyncStatus: replicationPolicy.?multiVmSyncStatus ?? 'Enable'
+      recoveryPointHistory: replicationPolicy.?recoveryPointHistory ?? 1440
```

```suggestion
      appConsistentFrequencyInMinutes: replicationPolicy.?appConsistentFrequencyInMinutes
      crashConsistentFrequencyInMinutes: replicationPolicy.?crashConsistentFrequencyInMinutes
      multiVmSyncStatus: replicationPolicy.?multiVmSyncStatus'
      recoveryPointHistory: replicationPolicy.?recoveryPointHistory
```

Redundant as anyways the default.